docs/docs/1.7.x/security/api-server-auth.md

```diff
@@ -228,8 +243,9 @@ If the signing key is compromised, we must rotate it and all the tokens that was
    ::::
 
 2. Regenerate user tokens
-   Create new user tokens. These tokens are automatically created with the signing key that’s assigned the highest serial number, so they’re created with the new signing key.
-   At this point, tokens signed by either new or old signing key are valid.
+
+   Create new user tokens. Tokens are always signed by the signing key with the highest serial number.
+   Now tokens signed by either new or old signing key are valid.
```

Comment:
Using "starting from now" to me more clearly indicates that the action just taken resulted in the change.

```suggestion
   Starting from now tokens signed by either new or old signing key are valid.
```